playbooks/vars/all.yml

```diff
@@ -4,6 +4,7 @@ gating_overrides_file: "/etc/openstack_deploy/user_zzz_gating_variables.yml"
 default_gating_overrides:
   maas_notification_plan: npTechnicalContactsEmail
   nova_virt_type: qemu
+  nova_cross_az_attach: True
```

Comment:
@mkam i was having a quick look at this before approving, and noticed that rpco explicitly disables `nova_cross_az_attach`.  I'm not entirely sure why that is but I think we best keep that behaviour so we're more closely testing what is getting deployed.  If tests fail because of this then we'll have to remove those tests from being run.  :(